Archiving/app/src/main/java/com/tokbox/android/tutorials/archiving/MainActivity.java

```diff
@@ -180,9 +228,14 @@ protected void onResume() {
 
     @Override
     public void onSessionConnectionDataReady(String apiKey, String sessionId, String token) {
+        Log.i(LOG_TAG, apiKey);
```

Comment:
The reason I prefer separate lines is that it makes it easier to view in the logs. Especially with our very long token, the other values can be hard to find, especially for someone that doesn't know what to expect.